apps/client/src/app/implementation-guide-wrapper/ignore-warnings/ignore-warnings.component.html

```diff
@@ -4,6 +4,6 @@
     <app-tooltip-icon tooltipKey="ig.ignoreWarnings"></app-tooltip-icon>
   </div>
   <div class="card-body">
-    <textarea class="form-control" rows="4" [(ngModel)]="intervalValue" [ngModelOptions]="{ updateOn: 'change' }"></textarea>
+    <textarea class="form-control" rows="4" [(ngModel)]="intervalValue" [ngModelOptions]="{ updateOn: 'change' }" [disabled]="readOnly"></textarea>
```

Comment:
The changes discussed earlier are _only_ for the purposes of the the publish template settings... Doesn't have anything to do with the `custom-menu` component or the `ignore-warnings` component. I think these changes to these two components can be reverted.